Description: Several improvements to the manpages.
 - Set an useful whatis-entry to panelctl's manpage.
 - Fix use of hyphen as minus sign.
Author: Alessio Treglia <alessio@debian.org>
Forwarded: no
---
 test/dvcont.1   |    2 +-
 test/panelctl.1 |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- libavc1394.orig/test/panelctl.1
+++ libavc1394/test/panelctl.1
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.36.
 .TH PANELCTL "1" "November 2009" "panelctl 0.2" "User Commands"
 .SH NAME
-panelctl \- manual page for panelctl 0.2
+panelctl \- remote control for set-top boxes
 .SH SYNOPSIS
 .B panelctl
 [\fIOPTION\fR...] \fI<channel|command>\fR
--- libavc1394.orig/test/dvcont.1
+++ libavc1394/test/dvcont.1
@@ -48,7 +48,7 @@ Tell the camera to play in reverse. If a
 .TP
 .B trickplay < -14 to +14 >
 Tell the camera to play at various speeds. 
-0 = forward play, -14  = fastest reverse, -8 = 1X reverse, -1 = slowest reverse,
+0 = forward play, \-14  = fastest reverse, \-8 = 1X reverse, \-1 = slowest reverse,
 1 = slowest forward, 8 = 1X forward, 14 = fastest forward.
 (Not supported by all cameras.)
 .TP
